imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -450,23 +450,23 @@ shcoarse:
 
 packet_version:
     <<: *default
-    CATDESC: Packet version - this will be incremented each time the format of the packet changes.
```

Comment:
I am pretty sure `CATDESC` is a required field but you may have set it to `' '` in your `default`. Later you might want to have short description in `CATDESC`

Reply:
Good to know, thanks! If I want to add `CATDESC`s for the ones that don't have them, I will have to come up with that myself most likely. But I will add a TODO for those and punt on that for a future PR.